### DISCUSSION TITLE
What's the difference useAtom in `jotai` and useAtom in `jotai/react`

### DISCUSSION THREAD

**Discussion by ThatXliner** [pmndrs/jotai#2701] [Q&A] [answered]
If I'm using `useAtom` from `jotai` in a Next.js app, is there any difference from that vs the `useAtom` in `jotai/react`?

The documentation doesn't answer my question

**Comment by dai-shi** [comment, marked as answer]
It's the same one:

https://github.com/pmndrs/jotai/blob/7d4da515040c0743907d283de145d7e81f0d6fef/src/index.ts#L2

It's mainly for library authors. For React app developers, using `useAtom` from `'jotai'` is totally fine.
